iothub/device/src/TransportSettings/IotHubClientAmqpSettings.cs

```diff
@@ -72,25 +73,19 @@ public TimeSpan OperationTimeout
         }
 
         /// <summary>
-        /// A keep-alive for the transport layer in sending ping/pong control frames when using web sockets.
-        /// </summary>
-        /// <seealso href="https://docs.microsoft.com/dotnet/api/system.net.websockets.clientwebsocketoptions.keepaliveinterval"/>
-        public TimeSpan? WebSocketKeepAlive { get; set; }
```

Comment:
Removed?

Reply:
Are we taking the path that if they want this stuff set, they provide the whole instance?

Reply:
And is that consistent with our similar situations, e.g., HttpClient?